tests/distributed/test_pyhccl.py

```diff
@@ -0,0 +1,124 @@
+#
```

Comment:
Add this file to https://github.com/vllm-project/vllm-ascend/blob/main/.github/workflows/vllm_ascend_test.yaml#L115(singlecard) 
 or https://github.com/vllm-project/vllm-ascend/blob/main/.github/workflows/vllm_ascend_test.yaml#L118 (multicard)to enable the CI test